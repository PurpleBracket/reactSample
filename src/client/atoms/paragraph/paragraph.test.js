/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import Paragraph from './paragraph'

describe('Headline', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<Paragraph>Bacon ipsum dolor amet brisket tenderloin shoulder sausage turducken hamburger, venison shankle tail prosciutto boudin salami ball tip. Chicken leberkas kielbasa ground round. Tri-tip ground round ham, ball tip filet mignon pancetta capicola. Alcatra meatloaf rump biltong. Kielbasa pork tri-tip meatloaf jowl, ground round sirloin hamburger pork chop shankle t-bone ribeye fatback frankfurter. Venison tenderloin ground round hamburger shankle flank turkey buffalo biltong meatloaf prosciutto pork chop corned beef cow. Spare ribs turducken chuck boudin landjaeger short loin, biltong frankfurter jowl porchetta.</Paragraph>)
    expect(wrapper).toMatchSnapshot()
  })
})
