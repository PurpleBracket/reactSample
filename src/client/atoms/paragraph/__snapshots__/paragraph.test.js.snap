// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should match snapshot 1`] = `
<p>
  Bacon ipsum dolor amet brisket tenderloin shoulder sausage turducken hamburger, venison shankle tail prosciutto boudin salami ball tip. Chicken leberkas kielbasa ground round. Tri-tip ground round ham, ball tip filet mignon pancetta capicola. Alcatra meatloaf rump biltong. Kielbasa pork tri-tip meatloaf jowl, ground round sirloin hamburger pork chop shankle t-bone ribeye fatback frankfurter. Venison tenderloin ground round hamburger shankle flank turkey buffalo biltong meatloaf prosciutto pork chop corned beef cow. Spare ribs turducken chuck boudin landjaeger short loin, biltong frankfurter jowl porchetta.
</p>
`;
